### ISSUE TITLE
[Suggestion] Set photo view without labels

### ISSUE THREAD

**Issue by Pheggas** [photoprism/photoprism#645]
It would be nice to view photos without that labeling. There would be setting for that in settings (something like checkbox).

**Comment by lastzero** [comment]
You can hide the title already if that's what you mean?

**Comment by Pheggas** [comment]
> You can hide the title already if that's what you mean?

I'm blind probably but, where can i set it? Didn't find anything related to it in settings.

**Comment by lastzero** [comment]
Tap the screen. But you mean completely without the titles and anything? That would need to be added in Settings.

**Comment by Pheggas** [comment]
I can achieve only photo view by clicking Toggle view

![image](https://user-images.githubusercontent.com/9994511/100073660-13cd3e00-2e3e-11eb-8ac3-860527c5bb95.png)

But i still can't customize it's preview size (make it bigger or smaller).

**Comment by lastzero** [comment]
Thought you're talking about the slideshow?

For search results, we offer these three views. Size is automatic depending on the size of your screen.

**Comment by Pheggas** [comment]
No, I'm not talking about slideshow. I'm talking about scaling of photos in this View.

For example:

I want to make the photos show bigger like here:
![image](https://user-images.githubusercontent.com/9994511/100229622-cf689d80-2f24-11eb-9a14-6a48d30e0388.png)

Or make it smaller like here:
![image](https://user-images.githubusercontent.com/9994511/100229713-f6bf6a80-2f24-11eb-9a1e-a37072327c81.png)

**Comment by lastzero** [comment]
The thumbnails have a specific, fixed sizes. You might show them bigger or smaller, but the quality would suffer. We don't render them dynamically based on the screen or your settings, would be way too slow.

**Comment by lastzero** [comment]
We could make the size for the Mosaic view configurable in a later release, but only based on what thumbnail sizes are available: 

https://docs.photoprism.org/developer-guide/library/thumbnails/#sizes

Square thumb sizes: 100px, 224px and 500px

**Comment by remisharrock** [comment]
That could enable to see more photos at once, especially on small screens. For example, comparing photoprism with google photos, on my mobile phone I can see a maximum of 18 photos with photoprism vs 45 with google photos ! I even remember Google Photos having a year view with a very compact layout and thumbnails (see the comparison of year view in Apple Photos vs Google Photos bellow). More than 120 photos could be displayed on the screen, I found this feature absolutely fantastic !
![Screenshot_20230423-200027](https://user-images.githubusercontent.com/8048877/233856997-5f15fcfd-f7c0-4920-adcb-4dbb8aacd7d3.png)
![Screenshot_20230423-200045](https://user-images.githubusercontent.com/8048877/233857060-9aae9baa-e259-4832-8631-250338edc5a9.png)
![Layout_2](https://user-images.githubusercontent.com/8048877/233858023-4a9d1235-c2d9-43c7-8d63-a1b54e2cecc1.jpg)
